Resources/Private/Templates/EnvironmentConsole.html

```
@@ -0,0 +1,8 @@
+<html xmlns:f="http://typo3.org/ns/TYPO3/CMS/Fluid/ViewHelpers"
+      data-namespace-typo3-fluid="true">
+
+<f:if condition="{script}">
+    <f:asset.script identifier="environment-indicator-console" nonce="{f:security.nonce(directive: 'script-src', scope: 'static')}">{script -> f:format.raw()}</f:asset.script>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)